lms/djangoapps/edraak_university/urls.py

```diff
@@ -15,7 +15,7 @@
        name='edraak_university_id',
     ),
     url(
-       r'^id/{}/success$'.format(settings.COURSE_ID_PATTERN),
+       r'^id/{}/(?P<is_student>.*)/success$'.format(settings.COURSE_ID_PATTERN),
```

Comment:
passing such a flag in the url is not best practice at all, your browser will appear as *.../id/course-v1:asdhajsndknal/True/success*.

Here you have two options to achieve what you are trying to achieve:
1 - Use a separate success page for instructor and student
2 - Use context data to pass variables to build the logic inside the HTML